vagrant/provisioning/roles/foia-analytical-reports/tasks/main.yml

```diff
@@ -112,6 +179,25 @@
       - "*~"
       - "*.zip"
   register: foia_dw_files
+  when: arkcase_version != "" and arkcase_version is version('2023.02', '<')
+
+- name: find project files
+  become: yes
+  become_user: pentaho
+  find: 
+    paths: 
+      - "{{ root_folder }}/install/pentaho/foia-reports-dw-2023.02.01-RC.01"
```

Comment:
All below, hardcoded. Please use variable

Reply:
Okay Martin The whole role was hardcoded will change that. my bad!